vcs/manageElements.py

```diff
@@ -1912,3 +1920,33 @@ def removeobject(obj):
         msg = 'This is not a template, graphics method, or secondary method object.'
         raise vcsError(msg)
     return msg
+
+
+def reset(vcstypes=None):
+    """Resets all vcs objects created by user
+
+    :Example:
+
+        .. doctest:: manageElements_reset
+
+            >>> a=vcs.init()
+            >>> iso=a.createisoline('dean') # Create an instance of an isoline object
+            >>> vcs.reset()
```

Comment:
Should `vcs.reset()` have the options for a `force=True` which will mean that ALL vcs created objects are purged, whereas the default behaviour (`force=False`) would leave objects that are currently references in place?

I wonder if we could also provide a short list of keywords: `isofill`, `meshfill` (or other objects - you guys know these better than me) so that only subtypes are purged?